docs/api_sphinx/requirements.txt

```diff
@@ -1,6 +1,6 @@
 myst_parser
-sphinx>=5
-sphinx-autobuild
-sphinx_rtd_theme==1.0.0
+sphinx
+sphinx_rtd_theme
+arize-phoenix-evals
```

Comment:
bit confused why this needs phoenix evals?

Reply:
What is the requirements file for here? Seems like any requirements should be inside `pyproject.toml`.

Reply:
I think is is a readthedoc build requirements file

Reply:
Readthedoc requires a requirements file for its build (and also doesn't accept toml files).